index.html

```diff
@@ -20,11 +20,23 @@
       <main class="parent">
         <header class="navbar">
           <nav>
-             <a href="#">Risk</a>
-          </nav>
-          <nav>
-            <i class="bi bi-list"></i>
+            <a href="#">Risk</a>
           </nav>
+          <div class="x-menu-icon">
+
+            <nav id="x-menu">
+              <i class="bi bi-list" id="menu-bar"></i>
+            </nav>
+            <!---popup navbar-->
+            
+            <i class="bi bi-x-lg" id="x-icon"></i>
+            <ul class="popup-list">
```

Comment:
- [ ] When the user clicks those options it should move to the correct part of the application, currently your app doesn't do that
https://www.awesomescreenshot.com/video/16626532?key=ed0d29cc9ef9c88242dfc9c6f8d63d30
- [ ] Also remember that after clicking the option and the app sending you to the correct part of the app, the popup should close